lib/topolys/schema/topolys.json

```diff
@@ -0,0 +1,135 @@
+{
```

Comment:
Why draft-04 (instead of 07)?

Reply:
I tried 07 but couldn’t get visual studio code or the Json-schema gem to work with it :-/

Reply:
Got it. But eventually moving up to 05, 06 or 07 will require some adaptation (e.g. $id, min/max constraints).

Reply:
Yes hopefully we can upgrade, I just couldn’t get it to work last night